File: TexasPoker/ExportProtocol/ExportProtocol/bin/Debug/cscode/responders/InitGameRspd.cs

//============================================
//--4>:
//    Exported by ExportProtocol
//
//    此代码为工具根据配置自动生成, 建议不要修改
//
//============================================

namespace Responders
{
	using System;
	using UnityEngine;
	using Socket;
	using Socket.Net;

	public class InitGameRspd : BaseRspd
	{
		public short mainMissionID;
		public InitGameRspd(ByteBuffer bytes)
			: base(bytes)
		{}
		public override void ReadData()
		{
			mainMissionID = m_Bytes.ReadShort();
		}
	}
}

File: TexasPoker/ExportProtocol/ExportProtocol/bin/Debug/javacode/responders/InitGameRspd.java

//============================================
//--4>:
//    Exported by ExportProtocol
//
//    此代码为工具根据配置自动生成, 建议不要修改
//
//============================================

package requesters;
import java.nio.ByteBuffer
import comm.BaseRspd;
import utils.JkTools;
import comm.Client;

public class InitGameRspd extends BaseRspd {
	private final short mainMissionID;
	public InitGameRspd(
			short mainMissionID
		) {
		super(10001);
		this.mainMissionID = mainMissionID;
	}
	@Override
	protected void push(Client cl) {
		super.push(cl);
		bytes.putShort(mainMissionID);
	}
}
